motr/client_internal.h

```diff
@@ -229,6 +229,9 @@ struct m0_op_idx {
 	struct m0_sm_group  *oi_sm_grp;
 	struct m0_ast_rc     oi_ar;
 
+	/* Minimum successful CAS requests. */
```

Comment:
Could you please use the more descriptive comment like
```
/**
  * Minimum number of successful CAS operations to treat
  * parent DIX operation as successful.
  * /
```
Current comment understand only those who are discussing that feature.

Reply:
Changed